src/jira/util/jwt.ts

```diff
@@ -184,6 +184,19 @@ export const verifySymmetricJwtTokenMiddleware = (secret: string, tokenType: Tok
 
 const ALLOWED_BASE_URLS = [BASE_URL]
 
+const isStagingTenant = (req: Request): boolean => {
+	let isStagingTenant = false;
+	const hostBaseUrl = _.get(req.body, "baseUrl");
+	if (hostBaseUrl) {
+		const host = new URL(hostBaseUrl).hostname;
```

Comment:
This needs to be wrapped in a try/catch in case the baseUrl isn't a URL.

Reply:
Ok, this one makes sense